airbyte-workers/src/main/java/io/airbyte/workers/temporal/discover/catalog/DiscoverCatalogActivityImpl.java

```diff
@@ -73,7 +78,10 @@ public ConnectorJobOutput run(final JobRunConfig jobRunConfig,
     final JsonNode fullConfig = secretsHydrator.hydrate(config.getConnectionConfiguration());
 
     final StandardDiscoverCatalogInput input = new StandardDiscoverCatalogInput()
-        .withConnectionConfiguration(fullConfig);
+        .withConnectionConfiguration(fullConfig)
```

Comment:
Do we need to version this in Temporal @pmossman ? Or is it ok to not do so because this is an activity impl and not a workflow?